.github/PULL_REQUEST_TEMPLATE/devconsole_pr.md

```diff
@@ -0,0 +1,29 @@
+
+# [JIRA-ID](add JIRA link): User Story / Defect / Task Title
+<!--For e.g [ODC-100](https://jira.coreos.com/browse/ODC-100): Title of user story / defect / task -->
+
+## Root cause
+<!--Briefly describe the root cause & analysis of the problem-->
+
+## Solution Description
+<!--Describe your code changes in detail and explain the solution-->
+
+- [ ] Is U/X approved?
+<!-- If designer review required, tag @openshift/team-devconsole-ux and add screenshots/gifs  -->
+
+- [ ] Covered Unit test cases / e2e test cases? 
+<!-- Were unit tests or E2E test recorded for this change, or was only manual testing applicable. If yes, attach report else state reason for not adding -->
+
+- [ ] Does this PR need regression testing of any feature / work flow?
+<!-- If the changes have bigger impact on any feature / work flow, do mention scope for regression testing -->
```

Comment:
This is a slippery slope... everything is a feature :) How do I know what we did 4 releases ago and what that ticket may be. Every change has a potential to change the way a feature works/worked.

What is the intent of this? Can you give an example?

Reply:
Intent was developer shall add regression testing scope (if any) to QA/reviewer. For example, if there are some critical changes in pipeline-actions then it might require whole Pipeline feature or multiple flows to be tested for regression.

Reply:
I wonder how useful this will be. Obviously if the reviewer knows about them, they have tested them. But I suppose forcing them to answer the question may help them realize they didn't test everything.

I think it's important for reviewers to incorporate this kind of regression testing as part of any review and not expect the reviewer to have covered everything. If we just trust the reviewer, we might as well not do reviews :P